library/tinkerpop/pom.xml

```diff
@@ -29,6 +29,8 @@
         <tinkerpop.version>3.7.1</tinkerpop.version>
         <google-guice.version>4.2.3</google-guice.version>
         <cucumber.version>7.15.0</cucumber.version>
+        <!-- To use a later version it requires us to update our scala version to v2.12.x -->
```

Comment:
Fair point this was more a reminder in case someone updated the version here as it will only crash at runtime not build time and the error message is not overly obvious. I could edit the message but I think its still worth noting here about the dependency.

Reply:
Could also use the enforcer plugin to prevent any accidental upgrades. This is more verbose than a comment but it can't be ignored.

Reply:
Probably not critical enough for the enforcer plugin just a small comment is likely enough, the latest version is only 1.0.4 anyways and development has mostly stopped so we're not likely to update anytime soon.